resources/xsd/discovery-server.xsd

```diff
@@ -201,7 +201,7 @@
     <xs:complexType name="writerTimesType">
         <xs:all minOccurs="0">
             <xs:element name="initialHeartbeatDelay" type="durationType" minOccurs="0"/>
-            <xs:element name="heartbeatPeriod" type="durationType" minOccurs="0"/>
+            <xs:element name="heartbeat_period" type="durationType" minOccurs="0"/>
```

Comment:
I am not sure if we should change the XML schema

Reply:
After internal discussion we decided to remove the file because it is not used